src/tests/electron/tests/android-setup-prompt-install-service.test.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    leftFooterButtonAutomationId,
+    rightFooterButtonAutomationId,
+} from 'electron/views/device-connect-view/components/android-setup/android-setup-step-layout';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { createApplication } from 'tests/electron/common/create-application';
+import { scanForAccessibilityIssues } from 'tests/electron/common/scan-for-accessibility-issues';
+import { AndroidSetupViewController } from 'tests/electron/common/view-controllers/android-setup-view-controller';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import {
+    commonAdbConfigs,
+    setupMockAdb,
+    simulateServiceNotInstalled,
+} from '../../miscellaneous/mock-adb/setup-mock-adb';
+
+describe('Android setup - prompt-install-service ', () => {
+    const defaultDeviceConfig = commonAdbConfigs['single-device'];
+    const installButtonId = 'install';
```

Comment:
can we import `installAutomationId` from `prompt-install-service-step.tsx` here?

Reply:
Fixed